### PR TITLE
add Chinese instructions

### PR DESCRIPTION
**Pull request by YunjieLi**
Added Chinese instructions (zh.json)

### CODE REVIEW

**Comment by YunjieLi** [comment]
I consulted a few people and feel 'slight left' and 'sharp left' is rarely used in Chinese and removed them as exceptions in some cases. This is only a MVP and other exceptions needs to be added in Chinese.

Would love to get extra eyes on it too. 

cc @freenerd

File: instructions/zh.json [hunk 1]
```diff
@@ -0,0 +1,296 @@
+{
+    "meta": {
+        "capitalizeFirstLetter": true
```
Comment (1ec5) [inline instructions/zh.json]: This can be `false` for Chinese.

File: instructions/zh.json [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+                "name": "乘坐{way_name}轮渡",
+                "destination": "乘坐开往{destination}的轮渡"
+            }
+        },
+        "arrive": {
+            "default": {
+                "default": "您已经到达您的{nth}个目的地"
+            },
+            "left": {
+                "default": "您已经到达您的{nth}个目的地, 在道路左侧"
```
Comment (1ec5) [inline instructions/zh.json]: Use the fullwidth comma, not an ASCII comma and space.

File: instructions/zh.json [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+            },
+            "straight": {
+                "default": "您已经到达您的{nth}个目的地, 在您正前方"
+            }
+        },
+        "continue": {
+            "default": {
+                "default": "继续{modifier}",
+                "name": "继续{modifier}，上{way_name}",
```
Comment (1ec5) [inline instructions/zh.json]: This seems like excessive comma usage for Chinese. Are you sure a comma is needed here?

File: instructions/zh.json (hunk 2 above)
Comment (YunjieLi) [inline instructions/zh.json]: Oh my apologies! Shall I change them in here or master? @freenerd

File: instructions/zh.json (hunk 3 above)
Comment (YunjieLi) [inline instructions/zh.json]: I personally think it helps to clarify the sentence, especially in the voice over. Also is what Google does.

File: instructions/zh.json (hunk 2 above)
Comment (1ec5) [inline instructions/zh.json]: In master or a new PR would be great. Thanks!

**Pull request merged**